### ISSUE TITLE
Missing license info

### ISSUE THREAD

**Issue by brunosaboia** [WordDocX/DocX#38]
Hello,

I could not find the license information for this project on GitHub. However, Codeplex states that it's Ms-PL (https://docx.codeplex.com/license). Is this information current and correct? Could you please include it on project's README.md? (I can create a pull request for that).

Thanks

**Comment by realvictorprm** [comment]
Yes providing license information is very important. Could you please add information about it?
